2021-06-28 16:55:55,473 - lab_planshet_2 - ERROR - Error detected
Traceback (most recent call last):
  File "C:\Users\Webdom16\PycharmProjects\TestsCovid\COVID_19_ 9880\Заполнение_лабораторных_планшетов\lab_planshet_2.py", line 31, in genlog
    self.lab_planshet()
  File "C:\Users\Webdom16\PycharmProjects\TestsCovid\COVID_19_ 9880\Заполнение_лабораторных_планшетов\lab_planshet_2.py", line 77, in lab_planshet
    driver.find_element_by_css_selector("#buttonsForm\:j_idt79")
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 598, in find_element_by_css_selector
    return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=91.0.4472.114)

